src/test/java/org/xstefank/check/AuxiliaryTestMethods.java

```diff
@@ -0,0 +1,25 @@
+package org.xstefank.check;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.xstefank.model.yaml.Format;
+import org.xstefank.model.yaml.FormatConfig;
+import org.xstefank.model.yaml.SkipPatterns;
+import java.io.File;
+import java.io.IOException;
+
+public class AuxiliaryTestMethods {
```

Comment:
do not name classes Auxiliary... the classname should clearly state what the class does, but auxiliary can have many meanings, traditionally similar classes are names as utils / utilities